docs/volunteer/index.md

```diff
@@ -5,4 +5,4 @@ Helping out a hackathon is a great way to get started with organising!
 
 ## Events looking for help
 
-(You can add to this list by submitting a PR [here](https://github.com/bahorn/hack.athon.uk)
+You can add to this list by submitting a PR [here](https://github.com/bahorn/hack.athon.uk)
```

Comment:
While we're here, can we update this URL to point to the new repo? https://github.com/HHEU/wiki rather than the old https://gihtub.com/bahorn/hack.ahon.uk